run.sh

```diff
@@ -12,4 +12,5 @@ source "$venvname/bin/activate"
 pip3 install -r requirements.txt
 
 export FLASK_APP=app.py
+export FLASK_DEBUG=1
```

Comment:
Didn't realize it was an option to add the debug flag here . I agree with this change, it keeps everything together in once place.